src/foam/nanos/auth/resetPassword/EmailView.js

```diff
@@ -129,6 +129,11 @@ foam.CLASS({
     {
       class: 'EMail',
       name: 'email'
+    },
+    {
+      class: 'String',
```

Comment:
This works, but usually when we want a view as a property I've seen a `foam.u2.ViewSpec` used. I think it's more flexible that way.

[Example](https://github.com/foam-framework/foam2/blob/master/src/foam/comics/DAOControllerView.js#L98)